client/src/containers/MapContainer.css

```diff
@@ -0,0 +1,49 @@
+.map-container {
+    color: #124e54;
+    margin-left: calc(-95vw / 2 + 450px / 2);
+    margin-top: -2.5em;
+    margin-bottom: -2.5em;
+    background: #FFFCF5;
+    box-shadow: 0px 14px 80px rgba(34, 35, 58, 0.2);
+    padding: 40px 55px 45px 55px;
+    border-radius: 15px;
+    position: absolute;
+    transition: all 0.3s;
+    width: 85vw;
+  }
+  
+  /* @media only screen and (max-width: 900px) {
+    .map-container {
+      margin-left: calc(-100vw / 2 + 450px / 2);
+      margin-right: 0;
+      width: 75vw;
+    }
+  } */
```

Comment:
If we're not using this block, can we delete it?

Reply:
Done!